llvmlite/binding/newpassmanagers.py

```diff
@@ -26,7 +26,7 @@ def __init__(self, ptr=None):
         super().__init__(ptr)
 
     def run(self, module, pb):
-        ffi.lib.LLVMPY_RunNewModulePassManager(self, pb, module)
+        ffi.lib.LLVMPY_RunNewModulePassManager(self, module, pb, pb._pic)
```

Comment:
I had previously not spotted that the arguments to `run()` were reversed compared to how they're passed into `LLVMPY_RunNewModulePassManager()` - making them consistent is a nice clean-up.

Reply:
However, one thing I don't quite get here - can the `PassBuilder._pic` object ever be out-of-sync with the `PIC` object attached to the underlying `PassBuilder` instance? I'm thinking: do we need to pass in `pb._pic` here, or could `LLVMPY_RunNewModulePassManager` just obtain it from `PB.getPassInstrumentationCallbacks()`?